test/e2e/tests/settings/clear-activity.spec.ts

```diff
@@ -30,7 +30,7 @@ describe('Clear account activity', function (this: Suite) {
         const activityList = new ActivityList(driver);
         await activityList.checkCompletedTxNumberDisplayedInActivity(2);
         await activityList.checkTxAction('Received', 1);
-        await activityList.checkTxAction('Sent', 2);
+        await activityList.checkTxAction('Sent', 1);
```

Comment:
This was incorrect. We only have 1 Sent tx imported in the wallet with fixtures.
Now we can see this error because the checkTxAction function has been fixed

<img width="1316" height="856" alt="Screenshot from 2025-09-08 11-30-04" src="https://github.com/user-attachments/assets/44e750af-8527-4422-ad2e-c4b51a96f17c" />